recipes/superblockify/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "superblockify" %}
+{% set version = "1.0.0rc6" %}
```

Comment:
This will have to be published as a prerelease software: https://conda-forge.org/docs/maintainer/knowledge_base/#pre-release-builds

Reply:
Thank you for highlighting this. I added the RC channel in the `conda_build_config.yaml`, see the last commit.